samples/subsys/tracing/app.overlay

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright (c) 2024 Tenstorrent AI ULC
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/ {
+    gpio_emul: gpio_emul {
```

Comment:
Indentation looks off a bit here. Probably 4 spaces that need to be replaced by a tab.